src/main/java/bio/terra/service/snapshotbuilder/utils/ConceptChildrenQueryBuilder.java

```diff
@@ -68,7 +68,9 @@ public Query buildConceptChildrenQuery(
     // COUNT(DISTINCT person_id)
     FieldVariable count = domainOccurrence.makeFieldVariable(PERSON_ID, "COUNT", "count", true);
 
-    List<SelectExpression> select = List.of(conceptName, conceptId, count);
+    List<SelectExpression> select =
+        List.of(
+            conceptName, conceptId, count, HierarchyQueryBuilder.hasChildrenExpression(concept));
```

Comment:
do we want another location like a utils class or something for this shared functionality? `HierarchyQueryBuilder.hasChildrenExpression(concept)`

Reply:
Maybe? Or maybe there's a way to combine `ConceptChildrenQueryBuilder` with `HierarchyQueryBuilder` in a future refactor, I think they have code in common. There is a general cleanup ticket for the query code https://broadworkbench.atlassian.net/browse/DC-823 so I could add this to that list.